query_execution/QueryExecutionTypedefs.hpp

```diff
@@ -66,6 +67,19 @@ enum QueryExecutionMessageType : message_type_id {
   kRebuildWorkOrderMessage,  // From Foreman to Worker.
   kRebuildWorkOrderCompleteMessage,  // From Worker to Foreman.
   kPoisonMessage,  // From the CLI shell to Foreman, then from Foreman to Workers.
+
```

Comment:
This list is intend to sort in the order of the life cycle of a query. So do the block locator messages, from registration, block adding / deleting / locating, to unregistration.